### ISSUE TITLE
Depending on published libraries with jsdependencies no longer passes on JS sources

### ISSUE THREAD

**Issue by Bathtor** [scala-js/jsdependencies#49]
I have recently come to notice that one of my Scala.js facade libraries no longer passes on its Javascript dependencies to downstream applications, causing them to break when calling code from the library that relies on those dependencies.

I made an [example project](https://github.com/Bathtor/transitive-js-dependencies) to showcase the setup that used to work for me, but no longer does.

I would appreciate some feedback if this expected behaviour and the way to set this up has merely changed at some point, or if this is indeed a bug in newer versions of SBT/Scala.js/jsdependencies (I'm not quite sure what upgrade introduced this change).

If it is considered a bug, I'm happy to put in some more time and try to track down exactly when it broke, if that would be helpful?

**Comment by sjrd** [comment]
That is expected. Your application also needs to use
```scala
.enablePlugins(JSDependenciesPlugin)
```
for it to read and process JavaScript dependencies from its transitive dependencies.

**Comment by Bathtor** [comment]
Ah, that makes sense. Before `sjs1.x` it was built-in so I didn't need to do that. And when I switched my libraries and applications to `sjs1.x` I never noticed that it broke, because they all still had the generated files from before in the target directory anyway. I only noticed now because I switched everything to Scala 2.13 yesterday, which creates a different output directory and there, of course, the dependency files weren't already lying around.

Thanks for the quick feedback!
